### ISSUE TITLE
Event Signal gets out of sync if several event are received in a single batch of data

### ISSUE THREAD

**Issue by tarag** [banjun/ReactiveSSE#2]
SSEvent Signal gets out of sync if several events are received from network connection in a single URLSessionDataDelegate didReceiveData call, because only one event is parsed from the data.

Correction in ReactiveOperator.swift:

change 

```
guard let parsed = try? EventStream.event.parse(AnyCollection(buffer)) else { return }
observer.send(value: SSEvent(parsed.output))
buffer = String(parsed.remainder)
```

to:

```
while let parsed = try? EventStream.event.parse(AnyCollection(buffer)) {
    observer.send(value: SSEvent(parsed.output))
    buffer = String(parsed.remainder)
 }
```

**Comment by banjun** [comment]
@tarag thank you pointing out! would you mind if you create a PR for that?

**Comment by banjun** [comment]
fixed by #3
